ingest/sqs/service/sqs_pool_tracker_test.go

```diff
@@ -0,0 +1,99 @@
+package service_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/osmosis-labs/osmosis/v24/app/apptesting"
+	"github.com/osmosis-labs/osmosis/v24/ingest/sqs/service"
+)
+
+type SQSServiceTestSuite struct {
+	apptesting.ConcentratedKeeperTestHelper
+}
+
+func TestSQSServiceTestSuite(t *testing.T) {
+	suite.Run(t, new(SQSServiceTestSuite))
+}
+
+// This is a sanity-check test ensuring that all pools are tracked and returned correctly.
+func (s *SQSServiceTestSuite) TestPoolTracker_Track() {
+	s.Setup()
+
+	allPools := s.PrepareAllSupportedPools()
+
+	poolTracker := service.NewPoolTracker()
+
+	////////// Concetrated
+
+	// Get the concentrated pool
+	concentratedPool, err := s.App.ConcentratedLiquidityKeeper.GetConcentratedPoolById(s.Ctx, allPools.ConcentratedPoolID)
+	s.Require().NoError(err)
+
+	// Track the concentrated pool
+	poolTracker.TrackConcentrated(concentratedPool)
+
+	// Get the concentrated pool
+	concentratedPools := poolTracker.GetConcentratedPools()
+	s.Require().Len(concentratedPools, 1)
```

Comment:
Can we also check if its equal to the concentratedPool on line 31, or is checking length good enough for this test

Reply:
IMO, this is sufficient